ui/appui-react/src/appui-react/cursor/cursormenu/CursorMenu.tsx

```diff
@@ -77,6 +77,7 @@ export class CursorPopupMenu extends React.PureComponent<
   public override componentWillUnmount() {
     this._componentUnmounting = true;
     SyncUiEventDispatcher.onSyncUiEvent.removeListener(this._handleSyncUiEvent);
+    this._componentUnmounting = false;
```

Comment:
It seems like this would pretty much cause the `this._componentUnmounting` to be quite useless if it is `false` only in the fraction of a second while we remove the listener, as the need for this would be for a race condition in the first place (the listener is triggered before we unmount, but executed after we unmounted, in which case, the value would still be `false` and cause the same issue as if it wasnt checked at all)

I think the `... = false` would make more sense in `componentDidMount` (and probably rename it to `this._isMounted` and invert the logic)

Reply:
Resolved